code/script.js

```diff
@@ -0,0 +1,170 @@
+//DOMs
+const MY_REPOS = 'https://api.github.com/users/anndimi/repos'
+const projectsContainer = document.getElementById("projects")
+const commitsContainer = document.getElementById("commits")
+const user = 'anndimi'
+const USER_URL = `https://api.github.com/users/${user}`
+const userContainer = document.getElementById("userProfile")
+
+const checkHTML = document.getElementById("HTML")
+const checkCSS = document.getElementById("CSS")
+const checkJavaScript = document.getElementById("JavaScript")
+let selectedLanguages = ['HTML', 'CSS', 'JavaScript']
+let chartDrawn = false
+
+//Create link images to social media
+const createImg = (url, alt) => {
+    let image = document.createElement("img")
+    image.src = url
+    image.alt = alt
+    return image
+}
+
+const linkedinIcon = createImg("./assets/linkedin-icon.png", "linkedin")
+const githubIcon = createImg("./assets/github-icon.png", "github")
+const facebookIcon = createImg("./assets/facebook-icon.png", "facebook")
+const branchIcon = createImg("./assets/branch-icon.png", "branch")
+const pushIcon = createImg("./assets/push-icon.png", "push")
+const repoIcon = createImg("./assets/repo-icon.png", "repo")
+
+let heroImgArray = ["./assets/img1.webp", "./assets/img2.webp", "./assets/img3.webp"],
```

Comment:
Ohhhh you managed to get different hero images!?! I didn't notice that until I was code! cool